src/components/controls/dates/DatePicker/DatePicker.tsx

```diff
@@ -114,7 +115,7 @@ interface StyledDatePickerProps {
 }
 
 const StyledDatePickerInput = styled(DatePickerInput, {
-  shouldForwardProp: (prop) => prop !== "hasLabel", // avoid passing `hasLabel` attribute to HTML element
+  shouldForwardProp: (prop) => isPropValid(prop) && prop !== "withCellSpacing", // avoid passing `hasLabel` attribute to HTML element
```

Comment:
Should be enough right? Since `hasLabel` and `willCellSpacing` are not valid HTML attributes?

```suggestion
  shouldForwardProp: (prop) => isPropValid(prop),
```